docs/basics.adoc

```diff
@@ -530,7 +530,26 @@ gremlin> :> g.V().values('name')
 ==>stephen
 ----
 
-The `:remote` command tells the console to configure a remote connection to Gremlin Server using the `conf/remote.yaml` file to connect.  That file points to a Gremlin Server instance running on `localhost`.  The `:>` is the "submit" command which sends the Gremlin on that line to the currently active remote.
+The `:remote` command tells the console to configure a remote connection to Gremlin Server using the `conf/remote.yaml` file to connect.  That file points to a Gremlin Server instance running on `localhost`.  The `:>` is the "submit" command which sends the Gremlin on that line to the currently active remote. By default the console is sessionless. If you wish to use variables with a remote connection you can either run the entire command on a single line or append session to the remote connect command.
```

Comment:
By default, the console is sessionless, meaning that each line sent in the console is interpreted as a single request. Multiple statements can be sent on a single line using a semicolon as the delimiter.

Alternately, you can establish a console with a session by specifying `session` when creating the connection. A console session allows you to reuse variables across several lines of input.

Also include references back to the TinkerPop docs on this topic:
- https://tinkerpop.apache.org/docs/current/reference/#console-sessions
- https://tinkerpop.apache.org/docs/current/reference/#sessions